monitors.yml

```diff
@@ -195,3 +195,18 @@ monitors:
         warning: 400000
     query: avg:zuul.pipeline.check_github.job.bonnyci_run_check.wait_time.max{host:zuul} > 800000
     type: metric alert
+  - message: |
+      {{#is_warning}}Only {{warn_threshold}} nodes available{{/is_warning}} 
+      {{#is_alert}} {{threshold}} nodes available!{{/is_alert}} 
+
+      @slack-bonny-dd-alerts
+    multi: false
+    name: Available Nodes
+    options:
+      notify_no_data: false
+      require_full_window: false
+      thresholds:
+        critical: 1
+        warning: 3
+    query: avg:nodepool.nodes.ready{host:nodepool} < 2
```

Comment:
Can we query for the sum of `ready` nodes and `used` nodes? The issue we are looking for is `building` nodes going directly to `deleting`, if we only query for `ready` nodes then we will get many false positives as we normally mark `ready` nodes `used` and a new node becomes `building`.